backend/apps/ifc_validation/tasks.py

```diff
@@ -330,16 +330,19 @@ def parse_info_subtask(self, prev_result, id, file_name, *args, **kwargs):
 
         task.mark_as_initiated()
 
-        # try to open IFC file (catch segfaults)
+        # try to open IFC file (only catch segfaults)
+        SEGFAULT_PROCESS_RETURNCODE = -11
         try:
             code = "import ifcopenshell; ifcopenshell.open('" + file_path + "')"
             check_program = [sys.executable, '-c', code, file_path]
             logger.debug(f'Command for {self.__qualname__}: {" ".join(check_program)}')
             subprocess.run(check_program, check=True)
 
         except subprocess.CalledProcessError as err:
-            task.mark_as_failed(err)
-            raise
+
+            if err.returncode == SEGFAULT_PROCESS_RETURNCODE:
```

Comment:
I would say the only possible return code for this is 0 unless there is a crash. Or 1 if there is a python exception. So I would rewrite to `< 0` which will catch more low-level C++ failure modes.

```suggestion
            if err.returncode < 0:
```